js/chatbot/static/ChatBot.svelte

```diff
@@ -42,6 +44,9 @@
 
 	let div: HTMLDivElement;
 	let autoscroll: boolean;
+	let images_dir: string = root_url
```

Comment:
@abidlabs Isn't this what `normalise_files` is meant to do?

Reply:
Good point. I just took a look though and it looks like `normalise_files` doesn't handle the string case fully. In particular, it doesn't attach the `/file=` or `/proxy` prefixes if the input is a string.

I'm not sure about the full implications of patching `normalise_files` so I just added the logic there myself.